qemu/tests/ansible_test.py

```diff
@@ -15,18 +16,31 @@ def run(test, params, env):
     1) Check ansible package exists
     2) Launch the guest
     3) Clone an ansible playbook repo
-    4) Generate the ansible-playbook command
-    5) Execute the playbook and verify the return status
+    4) Genreate inventory file
+    5) Generate the ansible-playbook command
+    6) Execute the playbook and verify the return status
 
     :param test: QEMU test object
     :param params: Dictionary with the test parameters
     :param env: Dictionary with test environment.
     """
+
+    def generate_invent_file(env):
```

Comment:
We don't need to generate the inventory file, use `",".join(ip_lst)` is enough.

Reply:
Ok. Updated.